frontend/packages/dev-console/src/components/topology/TopologyShortcuts.tsx

```diff
@@ -10,6 +10,9 @@ const TopologyShortcuts: React.ReactElement = (
     <Shortcut rightClick>Access context menu</Shortcut>
     <Shortcut click>View details in side panel</Shortcut>
     <Shortcut hover>Access create connector handle</Shortcut>
+    <Shortcut ctrl ctrlCmd keyName="F">
```

Comment:
Need to confirm with @openshift/team-devconsole-ux about the correct key. But the `F` key seems incorrect. There was a comment about this in the design doc.

Reply:
Yes I am aware about that but I am not sure until when it will be decided so I will be changing this according to the final UX in my PR for quick search component.

Reply:
That's fine. We can easily change this later. I just wanted to make sure that UX is working on this.